src/main/groovy/io/gatling/gradle/RecoverEnterprisePluginException.groovy

```diff
@@ -22,8 +22,13 @@ class RecoverEnterprisePluginException {
         } catch (UnsupportedJavaVersionException e) {
             throwTaskExecutionException(f.getThisObject(), """
                 |${e.getMessage()}
-                |In order to target the supported Java bytecode version, please configure targetCompatibility to Java ${e.supportedVersion} in your build configuration. (see https://docs.gradle.org/current/userguide/java_plugin.html#sec:java-extension)
-                |Or, reported class may come from your project dependencies, published targeting Java ${e.version}.
+                |In order to target the supported Java version, please use the following Gradle setting (requires Gradle 6.6 or later):
+                |compileJava {
+                |    options.release = ${e.supportedVersion}
```

Comment:
Since this can only be an issue for JDK 18+, it makes sense to me to use the newer `-release` option (which offers better guarantee of compatibility with the version of the Java standard library, not just the bytecode version), but it's only supported since Gradle 6.6. Then again, it probably doesn't make sense to be stuck on an old version of Gradle with the latest JDK anyway?

Thoughts?

Reply:
Agree